pkg/master/import_known_versions.go

```diff
@@ -40,4 +40,11 @@ import (
 	_ "k8s.io/kubernetes/pkg/apis/scheduling/install"
 	_ "k8s.io/kubernetes/pkg/apis/settings/install"
 	_ "k8s.io/kubernetes/pkg/apis/storage/install"
+
+	apiserverinternal "k8s.io/apiserver/pkg/apis/apiserverinternal/install"
+	"k8s.io/kubernetes/pkg/api/legacyscheme"
 )
+
+func init() {
+	apiserverinternal.Install(legacyscheme.Scheme)
+}
```

Comment:
this seems a bit strange... the internal types live in k8s.io/apiserver, but are stitched into the legacy scheme and served by kube-apiserver? @deads2k, does this make sense to you?

Reply:
> this seems a bit strange... the internal types live in k8s.io/apiserver, but are stitched into the legacy scheme and served by kube-apiserver? @deads2k, does this make sense to you?

I didn't notice.  While I think it would be better to include the the internal types in kube-apiserver, I'm not very upset by having them in k8s.io/apiserver.

Reply:
serving the API in kube-apiserver was based on https://github.com/kubernetes/kubernetes/pull/84016#discussion_r384750723

I think it makes more sense to put the internal types in kube-apiserver (k8s.io/kubernetes/pkg/apis) and put the versioned types in k8s.io/api, which will also simplify the code generation. Can we have a followup PR to move the types?

Reply:
Sent https://github.com/kubernetes/kubernetes/pull/93087 to move the types